.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Logs**
+If applicable, add the link to a pastebin with the output of `noisetorch -log` and `pactl list short`.
+
+**Desktop (please complete the following information):**
+ - Distribution [e.g. Ubuntu 22.04, Arch]: 
```

Comment:
```suggestion
 - Distribution [e.g. Ubuntu 22.04, Arch]: 
 - DE [e.g. Plasma 5.24.5, GNOME 42.0]: 
```
I think it's worth including DE here as well, in the past it sounds like there were/are some bugs that are DE specific (DWM in one case). Even though a lot of the time we can derive DE from distro this will probably simplify things (I don't necessarily remember when Ubuntu switched from Unity to GNOME, etc).